translib/transformer/transformer.go

```diff
@@ -115,58 +112,73 @@ func loadYangModules(files ...string) error {
 		yang.AddPath(expanded...)
 	}
 
-	ms := yang.NewModules()
+	ygSchema, err := ocbinds.GetSchema()
+	if err != nil {
+		panic("Error in getting the schema: " + err.Error())
+	}
 
-	for _, name := range files {
-		if err := ms.Read(name); err != nil {
-			fmt.Fprintln(os.Stderr, err)
-			continue
+	modProfiles := make(map[string]ModProfile)
+	rootSchema := ygSchema.RootSchema()
+	for _, schema := range rootSchema.Dir {
+		if _, found := schema.Annotation["modulename"]; found {
+			m := schema.Annotation["modulename"].(string)
+			for _, fn := range files {
+				f := strings.Split(fn, ".yang")
+				if m == f[0] {
+					modProfiles[m] = schema.Annotation
+					break
+				}
+			}
 		}
 	}
 
-	// Process the Yang files
-	reportIfError(ms.Process())
-
-	// Keep track of the top level modules we read in.
-	// Those are the only modules we want to print below.
-	mods := map[string]*yang.Module{}
-	var names []string
-
-	for _, m := range ms.Modules {
-		if mods[m.Name] == nil {
-			mods[m.Name] = m
-			names = append(names, m.Name)
+	annotMs := yang.NewModules()
+	for _, name := range files {
+		if strings.Contains(name, "-annot") {
+			if err := annotMs.Read(name); err != nil {
+				fmt.Fprintln(os.Stderr, err)
```

Comment:
Here and elsewhere, please use log.V(lvl).Infof(...)

Reply:
Addressed. Used internal wrapper that we already had for verbosity level logs.